### PR TITLE
Allow customers to delete payment cards.

### PR DESCRIPTION
**Pull request by amyboyd**
Don't merge - the tests haven't been run. Waiting on the backend API, then I'll add the JS lib method `UsersFactory.deletePaymentCard`.

### CODE REVIEW

File: tests/e2e/customer/AccountDetails.js [hunk 1]
```diff
@@ -80,4 +74,48 @@ describe('Customer delivery addresses', function() {
         expect(firstParagraphText).toContain('Aperture Science');
         expect(firstParagraphText).toContain('Customer');
     });
+
+    describe('payment cards', function() {
+        var cards;
+        var NUMBER_OF_CARDS_IN_FIXRURES = 1;
```
Comment (lucygilbert) [inline tests/e2e/customer/AccountDetails.js]: Fixrures?

Comment (amyboyd) [inline tests/e2e/customer/AccountDetails.js]: It's the new way of spelling `fixtures`

Comment (lucygilbert) [inline tests/e2e/customer/AccountDetails.js]: You're fired.

**Comment by amyboyd** [comment]
Now ready for review

image

**Pull request merged**